### PR TITLE
Create alerts

### PR DESCRIPTION
**Pull request by Ozzorany**
image

### CODE REVIEW

**Review by TalShperling — APPROVED** [review]

File: src/app/registration/components/sign-in/sign-in.component.ts [hunk 1]
```diff
@@ -7,20 +7,23 @@ import firebase from 'firebase';
...
 import {UsersService} from '../../../services/users/users.service';
+import {AlertsService} from '../../../services/alerts/alerts.service';
 
 @Component({
   selector: 'app-sign-in',
   templateUrl: './sign-in.component.html',
   styleUrls: ['./sign-in.component.scss']
 })
 export class SignInComponent implements OnInit {
+  private errorAlert: string = 'The username or password is incorrect';
```
Comment (yarivg) [inline src/app/registration/components/sign-in/sign-in.component.ts]: - [x] rename

File: tslint.json [hunk 2]
```diff
@@ -69,7 +69,7 @@
     ],
     "no-empty": false,
     "no-inferrable-types": [
-      true,
+      false,
```
Comment (yarivg) [inline tslint.json:72]: why?

**Review by yarivg — CHANGES_REQUESTED** [review]

File: tslint.json (hunk 2 above)
Comment (Ozzorany) [reply tslint.json:72]: If it's true, it complains about types for variables in some cases.
For example **message: string = ' '** will throw tslint error, and would recommend to remove the type string.

File: src/app/registration/components/sign-in/sign-in.component.ts (hunk 1 above)
Comment (Ozzorany) [reply src/app/registration/components/sign-in/sign-in.component.ts]: done

**Comment by Ozzorany** [comment]
@LiorVais @yarivg 
Recheck please, I have merged develop changes & added changes of my own

**Review by yarivg — APPROVED** [review]

**Pull request merged**